### PR TITLE
Check for the sanity of CRD files

### PR DESCRIPTION
**Pull request by baijum**
Resolves: #40

The latest CRD files under the manifests directory and the CRD
files under deploy/crds should match.

### CODE REVIEW

**Comment by corinnekrych** [comment]
@baijum I'm really buying into this sanity check until we got sth even better like only one source of truth. 😍 

I think it would be good to add another check (more work since it's not just file verification) to check the content of `deploy/role.yaml` and compare it to `clusterPermission` section of `devopsconsole-operator.v0.1.0.clusterserviceversion.yaml`

mmm since we'll got on per version, long run we need some sort of code generation.

**Comment by corinnekrych** [comment]
@baijum I've started a discussion in here: https://coreos.slack.com/archives/C3VS0LV41/p1553697141394800
and https://coreos.slack.com/archives/C3VS0LV41/p1553697786396900?thread_ts=1553697687.396600&cid=C3VS0LV41
and it seems we got:
```
operator-sdk olm-catalog gen-csv --csv-version 0.2.0`
```
that will generate `deploy\devo[sconsole-oprator.csv.yaml` and `deploy/_generated.concat_crd.yaml`
So I think a better approach will be to have those files generated as part of the build. wdyt?

**Comment by baijum** [comment]
> @baijum I've started a discussion in here: https://coreos.slack.com/archives/C3VS0LV41/p1553697141394800
> and https://coreos.slack.com/archives/C3VS0LV41/p1553697786396900?thread_ts=1553697687.396600&cid=C3VS0LV41

Thanks for bringing up this into the operator framework developers.

> and it seems we got:
> 
> ```
> operator-sdk olm-catalog gen-csv --csv-version 0.2.0`
> ```
> that will generate `deploy\devo[sconsole-oprator.csv.yaml` and `deploy/_generated.concat_crd.yaml`
> So I think a better approach will be to have those files generated as part of the build. wdyt?

Once we have good test coverage, we can evaluate those options.

File: test/sanitychecks/crd_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	Channels []struct {
+		CurrentCSV string `yaml:"currentCSV"`
+		Name       string `yaml:"name"`
+	} `yaml:"channels"`
+	PackageName string `yaml:"packageName"`
+}
+
+func TestLatestCRDFiles(t *testing.T) {
+	curdir, err := os.Getwd()
+	if err != nil {
```
Comment (kwk) [inline test/sanitychecks/crd_test.go]: What about `require.NoError(t, err)`?

Comment (baijum) [reply test/sanitychecks/crd_test.go]: Done. 6a786427ccd6fafb9eefa7a480182b7830135842

**Pull request merged**